evennia/scripts/globalhandler.py

```diff
@@ -0,0 +1,45 @@
+from django.conf import settings
+from evennia.utils.utils import class_from_module
+
+
+class GlobalHandler(object):
+    """
+    Simple Handler object loaded by the Evennia API to contain and manage a game's Global Scripts.
+
+    This is accessed like a dictionary.
+
+    Example:
+        import evennia
+        evennia.GLOBAL_HANDLER['key']
+    """
+
+    def __init__(self):
+        self.typeclass_storage = dict()
+        self.script_storage = dict()
+        for k, v in settings.GLOBAL_SCRIPTS.items():
+            self.typeclass_storage[k] = class_from_module(v)
+        for k, v in self.typeclass_storage.items():
+            found = v.objects.filter_family(db_key=k).first()
```

Comment:
If you have a good idea here, all the more power to you. How does this system know how many instances of the script you want to run though? And if there are multiple scripts of the same base name it's no longer an easy index to get to the one script...

Reply:
That's not the issue - it's the `filter_family` that concerns me. This query will catch the first script of that name in the entire family of typeclasses. I don't think you'll have more than one instance of a given Script + name, but this appears to restrict the name across the entire family of typeclasses.

Reply:
Doesn't filter() by itself return ANYTHING in ScriptDB though?

Reply:
`ScriptDB.objects.filter()` will return all Scripts (same as `ScriptDB.objects.filter_family` will, since the `ScriptDB` is literally the parent of every script). In contrast, `Typeclass.objects.filter()` will only return scripts of that Typeclass, whereas `Typeclass.objects.filter_family` will match for that Typeclass and also all its children. 
You have the typeclass (specified from settings) already, so it seems you could query very specifically for the script you want to be running ...

Reply:
Ah, I forgot that .filter() is overloaded this way in Evennia.